apps/design-system/src/views/pull-request-compare/pull-request-compare.tsx

```diff
@@ -0,0 +1,66 @@
+import { FC, useCallback } from 'react'
+
+import { HeaderProps, PullRequestCompare, SandboxPullRequestCompareProps } from '@harnessio/ui/views'
+
+import { noop, useTranslationsStore } from '../../utils.ts'
+import { repoBranchListStore } from './repo-branch-store.ts'
+import { repoCommitStore } from './repo-commit-store.ts'
+
+const PullRequestCompareWrapper: FC<Partial<SandboxPullRequestCompareProps>> = props => {
+  const useRepoBranchListStore = useCallback(
+    () => ({
+      ...repoBranchListStore,
+      setPage: noop
+    }),
+    []
+  )
+  const useRepoCommitsListStore = useCallback(
+    () => ({
+      ...repoCommitStore,
+      setPage: noop
+    }),
+    []
+  )
+
+  return (
+    <PullRequestCompare
+      onFormSubmit={noop}
+      onFormDraftSubmit={noop}
+      onFormCancel={noop}
+      apiError={null}
+      isSuccess={false}
+      onSelectCommit={noop}
+      selectBranch={noop}
+      targetBranch={{ name: 'main', sha: 'sha' }}
+      sourceBranch={{ name: 'main', sha: 'sha' }}
+      diffData={[
+        {
+          text: 'bot.txt',
+          data: 'diff --git a/bot.txt b/bot.txt\nnew file mode 100644\nindex 0000000000000000000000000000000000000000..0aa158abf5a4f0c5d2a5697866f14601881647b3\n--- /dev/null\n+++ b/bot.txt\n@@ -0,0 +1 @@\n+bot5555\n\\ No newline at end of file\n',
+          title: 'bot.txt',
+          lang: 'txt',
+          addedLines: 1,
+          removedLines: 0
+        } as HeaderProps
```

Comment:
You can kill this now
```suggestion
        }
```

Reply:
done